docker/test/integration/features/hashcontent.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Adding hash value for a FlowFile using HashContent
+  In order to calculate a hash value for a FlowFile
+  As a user of MiNiFi
+  I need to have HashContent Processor
+
+Background:
+  Given the content of "/tmp/output" is monitored
+
+Scenario: HashContent adds hash attribute to flowfiles
+  Given a GetFile processor with "Input Directory" property set to "/tmp/input" 
```

Comment:
this file was included in this PR by accident, I think

Reply:
Yeah my bad. Sorry.